zerver/forms.py

```diff
@@ -78,6 +87,25 @@ def clean_email(self):
             return data
         raise ValidationError(mark_safe(SIGNUP_STRING))
 
+class RealmCreationForm(forms.Form):
+    # This form determines whether users can
+    # create a new realm. Be careful when modifying the
+    # validators.
+    email = forms.EmailField(validators=[is_inactive,])
+
+    def __init__(self, *args, **kwargs):
+        self.domain = kwargs.get("domain")
+        if "domain" in kwargs:
+            del kwargs["domain"]
+        super(RealmCreationForm, self).__init__(*args, **kwargs)
+
+    def clean_email(self):
```

Comment:
Hmm, that's reasonable, but probably we want to throw an error even if there is an "inactive" user with that email in another realm, so that's quite the right check.